Correct error in Thales summary. Add Anaximander, Anaximenes, and Pythagoras.

ashelkovnykov opened the pull request.

ashelkovnykov commented on _posts/philosophers/2020-06-26-anaximander-of-miletus.md:
```diff
@@ -0,0 +1,18 @@
+tags: []
+summary:
+author: AS
+---
```
@philozophy @joshvfleming 

I copied the entry pasted for Thales of Miletus. However, I have some comments about this header:
- Is the date necessary? As I understand, this should be like a short encyclopaedia entry, not a regular blog post.
- What is the "summary" field for?
- Is the author tag necessary? Since you guys are editing my entries, I assumed that this would be considered a group summary: everyone working on the blog agrees upon the summary. Furthermore, I'd rather force someone to have to crawl through the Git logs to see specifically what I wrote.

philozophy replied:
Feel free to skip the summary. I would keep the date, just for internal records. It's also nice to know if the entry is revised, I also keep track of that. The Stanford Encyclopedia of Philosophy also does that. Maybe not critical for philosophers, but let's keep the date, if that's ok. OK to remove the author too, but I would like to sign my own entries that I publish in the main blog.

ashelkovnykov commented:
@philozophy @joshvfleming 
- Modified tags to be: `title`, `date`, `revised`, `category`, `tags`
- Added tags
- Updated date to reflect true completion date
- Modified Git commit message to fit desired pattern for repo
